src/main/java/com/conveyal/datatools/manager/controllers/api/ProjectController.java

```diff
@@ -216,14 +220,28 @@ private static Project checkProjectPermissions(Request req, Project project, Str
      * to getFeedDownloadCredentials with the project ID to obtain either temporary S3 credentials or a download token
      * (depending on application configuration "application.data.use_s3_storage") to download the zip file.
      */
-    private static String downloadMergedFeed(Request req, Response res) {
+    static String downloadMergedFeed(Request req, Response res) {
```

Comment:
This method could be renamed to indicate it's not really downloading a merged feed.